conf/nutch-default.xml

```diff
@@ -1136,6 +1136,28 @@
 <!-- parser properties -->
 
 <property>
+  <name>parser.modelfilter.trainfile</name>
+  <value>tweets-train.tsv</value>
+  <description>
```

Comment:
These all need descriptions. If they do not have descriptions then no one has a clue what they do.